From 9eec574f85800cb4e451210c27127fcab0810bc8 Mon Sep 17 00:00:00 2001
From: GloriousEggroll <gloriouseggroll@gmail.com>
Date: Fri, 28 Jun 2024 15:11:03 -0600
Subject: [PATCH] fixup service unit error:

Unit /usr/lib/systemd/user/yumex-updater-systray.service is added as a dependency to a non-existent unit graphical.target.
---
 data/service/yumex-updater-systray.service | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/data/service/yumex-updater-systray.service b/data/service/yumex-updater-systray.service
index 41b66e8..4d989fe 100644
--- a/data/service/yumex-updater-systray.service
+++ b/data/service/yumex-updater-systray.service
@@ -1,6 +1,7 @@
 [Unit]
 Description=Service to monitor and manage system updates.
-After=graphical.target
+PartOf=graphical-session.target
+After=graphical-session.target
 
 [Service]
 Type=simple
@@ -11,4 +12,5 @@ Environment=DISPLAY=:0
 Environment=XAUTHORITY=%h/.Xauthority
 
 [Install]
-WantedBy=graphical.target
+WantedBy=graphical-session.target
+
-- 
2.45.2

